lib/ui/platform_dispatcher.dart

```diff
@@ -168,6 +168,23 @@ class PlatformDispatcher {
   // A map of opaque platform view identifiers to view configurations.
   final Map<Object, ViewConfiguration> _viewConfigurations = <Object, ViewConfiguration>{};
 
+  /// The implicit [FlutterView] provided by the platform, if any.
+  ///
+  /// This is used to bootstrap the framework for applications where the
+  /// platform provides a view, such as applications designed for
+  /// single-display mobile devices.
+  ///
+  /// While the [FlutterView]'s properties may change, this reference is
+  /// guaranteed to never change.
+  ///
+  /// See also:
+  ///
+  /// * [View.of], for accessing the current view.
+  FlutterView? get implicitView => _implicitViewEnabled() ? _views[0] : null;
+
+  @Native<Handle Function()>(symbol: 'PlatformConfigurationNativeApi::ImplicitViewEnabled')
+  external static bool _implicitViewEnabled();
```

Comment:
Wondering if this maybe should just return the implicit view ID (or null) to avoid that we have to hard-code the magic number here and in platform_configuration.cc.

Reply:
I had this approach originally but I switched to a boolean instead as we don't have any scenarios where the implicit view will have a non-zero ID. I don't feel strongly either ways though, so let me know if you have a preference.

Reply:
No preference, really. It's all private and internal anyways, so easy to change if it has to.

Reply:
Sounds good. I'll leave this open so others can chime in :)